tests/RobotFramework/tests/tedge_flows/tedge_flows.robot

```diff
@@ -116,31 +116,38 @@ Consuming messages from a process stdout, periodically
 
 Consuming messages from the tail of file
     # Assuming the flow tail-named-pipe.toml has been properly installed
+    Install Flow    tail-named-pipe.toml
```

Comment:
The comment in the previous line isn't valid anymore. Same applicable to the rest of the tests as well.